mmeds/validate.py

```diff
@@ -621,6 +645,20 @@ def check_study_name(self):
             self.errors.append(f'-1\t-1\tStudy Name Error: The study name in the metadata ({df_study_name})' +
                                f' does not match the name provided for this upload ({self.study_name})')
 
+    def get_subjects(self):
+        Logger.debug("getting subjects")
+        subjects = pd.DataFrame()
+        if self.subject_type == 'human' and 'Subjects' in self.df.keys():
+            subjects = self.df['Subjects']
+        elif self.subject_type == 'animal' and 'AnimalSubjects' in self.df.keys():
+            subjects = self.df['AnimalSubjects']
+        elif self.subject_type == 'mixed':
+            Logger.debug(f"subjects:\n{self.df['Subjects']}\nanimals:\n{self.df['AnimalSubjects']}")
```

Comment:
we don't do that in validate.py because we want all errors and warnings to be passed to the user, rather than to throw an error. Adding those user-passed errors instead